examples/graphics/clustered-lighting.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>PlayCanvas Clustered Lighting</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="icon" type="image/png" href="../playcanvas-favicon.png" />
+    <script src="../../build/playcanvas.prf.js"></script>
+    <script src="../../build/playcanvas-extras.js"></script>
+    <script src="../assets/scripts/asset-loader.js"></script>
+    <style>
+        body {
+            margin: 0;
+            overflow-y: hidden;
+        }
+    </style>
+</head>
+
+<body>
+    <!-- The canvas element -->
+    <canvas id="application-canvas"></canvas>
+
+    <!-- The script -->
+    <script>
+        var canvas = document.getElementById("application-canvas");
+
+        // Create the application and start the update loop
+        var app = new pc.Application(canvas);
+
+        // A list of assets that need to be loaded
+        var assetManifest = {
+            normal: {
+                type: "texture",
+                url: "../assets/textures/normal-map.png"
+            }
+        };
+
+        // Load all assets and then run the example
+        loadManifestAssets(app, assetManifest, function () {
+            run();
+        });
+
+        var pointLightList = [];
+        var spotLightList = [];
+        var dirLight = null;
+
+        function run() {
+
+            app.start();
+
+            // enabled clustered lighting. This is a temporary API and will change in the future
+            pc.WorldClusters.enabled = true;
```

Comment:
I'd rather not expose the `WorldClusters` class. Maybe have `LayerComposition.clusteredLightingEnabled` instead?

Reply:
done, thanks